### PR TITLE
Added my blog and other sources 

### LINKED ISSUE #18: Add a resource
This is the easiest green square you will get today!

All you need to do is add a repo and name resource to the readme that falls under folks identifying as women, transgender people, nonbinary people, and other underrepresented genders leading open source projects and companies, serving as maintainers, or otherwise supporting Open Source.

See [How to Contribute](https://github.com/amandamartin-dev/women-led-open-source/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by CBID2**
I was inspired by Snipe's advice and added my blog and some other sources 
Fixes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ Click on the [CONTRIBUTING.md](https://github.com/amandamartin-dev/women-led-ope
 [Women in Technology](https://github.com/womenintechnology) - We bring together women whose goal is to support each other in developing professional competencies, preparing for leadership roles, building a network of contacts, and increasing visibility in IT. [Women in Technology website](https://www.womenintechnology.pl/)
 
 ## Content
+<<<<<<< HEAD
+[Chrissy's Open Source Survival Guide](https://chrissycodes.hashnode.dev/) - Blog - Whether you are a coding newbie or an experienced developer looking to keep your skills sharp, Chrissy's series will help you navigate the fun and whimsical world of open source projects.
+[Open Source Fridays](https://youtube.com/playlist?list=PL0lo9MOBetEFmtstItnKlhJJVmMghxc0P) - YouTube - Every Friday, Rizel Scarlett from GitHub interviews open source founders about the origin of their project, how it's used, and how people can contribute to them.  
+[Open Source Stories](https://www.opensourcestories.org/) - Podcast - The Open Source Stories project celebrates this expanded view on what is counted and who represents open source by collecting narratives from creators, contributors, and consumers alike.
```
Comment (amandamartin-dev) [inline README.md:93]: Hey Chrissy! Great additions, but you duplicated an existing one. Double check the changes but otherwise lgtm

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CBID2) [reply README.md:93]: Hey Amanda! :) I checked and didn't any duplicates.

Comment (amandamartin-dev) [reply README.md:93]: weird, on my end it looks like you re-added OSS podcast...but I"ll merge and edit if it's dubbed.  Thanks!

**Review by amandamartin-dev — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (CBID2) [reply README.md:93]: Ohh I know what it is. I added Rizel's series. It's called "Open Source Fridays". That's probably why you see the duplicate.

**Pull request merged**